### PR TITLE
Add footer tweaks

### PR DESCRIPTION
**Pull request by angel**
This accomplishes most of the changes for the footer component to match v2 in this issue: #574 and enlarged the social icons to fix this issue: #346.

I'm just missing making the topics in the mobile big footer to be styled like our accordions: 
image

Also, when I click on a social media icon I get this white line:
image
Not sure how to fix that.

### CODE REVIEW

File: assets/_scss/components/_footer.scss [hunk 1]
```diff
@@ -161,11 +173,23 @@ li.usa-footer-primary-content {
...
+  color: $color-white;
   padding: {
     top: 3rem;
     bottom: 3rem;
   }
+
+  a {
+    color: $color-white;
+
+    &:hover {
```
Comment (kaelig) [inline assets/_scss/components/_footer.scss]: Any reason why there is no :focus state defined?

**Comment by kaelig** [comment]
You might want to optimize the SVG and PNG icons using services such as ImageOptim (https://imageoptim.com/) and SVGO or https://jakearchibald.github.io/svgomg/ to save some precious bytes (unless they're optimised at deploy time using a build step I would have missed?).

**Comment by juliaelman** [comment]
On hold until WDS 2.0 roadmap is completed.

File: assets/_scss/components/_footer.scss (hunk 1 above)
Comment (maya) [inline assets/_scss/components/_footer.scss]: Sorry this is so late @kaelig -  :focus state for links is defined in `src/elements/_typography.scss`.

**Comment by maya** [comment]
This will need additional research into the new footer designs for #574 before we can merge this, so closing for now. #346 will remain open so we can follow that up separately.

**Pull request closed without merging**